integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLiftAndShiftFromOnPremDomain.java

```diff
@@ -0,0 +1,481 @@
+  private static final String DISCOVER_DOMAIN_OUTPUT_DIR = "wkomodelfilesdir";
+  private static final String DOMAIN_TEMP_DIR = LIFT_AND_SHIFT_WORK_DIR + "/" + ON_PREM_DOMAIN;
+  private static final String DOMAIN_SRC_DIR = RESOURCE_DIR + "/" + ON_PREM_DOMAIN;
+  private static final String WKO_IMAGE_FILES = LIFT_AND_SHIFT_WORK_DIR + "/u01/" + DISCOVER_DOMAIN_OUTPUT_DIR;
+  private static final String WKO_IMAGE_NAME = "onprem_to_wko_image";
+  private static final String WKO_DOMAIN_YAML = "wko-domain.yaml";
+  private static final String BUILD_SCRIPT = "discover_domain.sh";
+  private static final Path BUILD_SCRIPT_SOURCE_PATH = Paths.get(RESOURCE_DIR, "bash-scripts", BUILD_SCRIPT);
+  private static final String domainUid = "onprem-domain";
+  private static int managedServerPort = 8001;
+  private static final String adminServerPort = "7001";
+  private static final String appPath = "opdemo/index.jsp";
+  private static String imageName = null;
+  private static LoggingFacade logger = null;
+  private Path zipFile;
+
+  private static HelmParams traefikHelmParams = null;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(6, MINUTES).await();
+
+    // create a reusable quick retry policy
+    withQuickRetryPolicy = with().pollDelay(1, SECONDS)
+        .and().with().pollInterval(2, SECONDS)
+        .atMost(15, SECONDS).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique Voyager namespace
+    logger.info("Get a unique namespace for Voyager");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    traefikNamespace = namespaces.get(1);
+
+    logger.info("Creating unique namespace for Domain");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domainNamespace = namespaces.get(2);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // install and verify Traefik
+    logger.info("Installing Traefik controller using helm");
+    traefikHelmParams = installAndVerifyTraefik(traefikNamespace, 0, 0);
+
+  }
+
+  // Create a MiiDomain from an on prem domain.
+  // This test first uses WDT DiscoverDomain tool on an on-prem domain. This tool when used with
+  // target option wko, will create the necessary wdt model file, properties file, an archive file
+  // and a domain yaml file. The test then use the resulting model file to create an MiiDomain
+  @Test
```

Comment:
Please make it as javadoc as opposed to comments

Reply:
Done

Reply:
really done now.